environment.yml

```diff
@@ -1,21 +1,337 @@
+  - statsmodels=0.14.0=py310h278f3c1_1
+  - tbb=2021.9.0=hf52228f_0
+  - terminado=0.17.1=pyh41d4057_0
+  - threadpoolctl=3.1.0=pyh8a188c0_0
+  - tinycss2=1.2.1=pyhd8ed1ab_0
+  - tk=8.6.12=h27826a3_0
+  - toml=0.10.2=pyhd8ed1ab_0
+  - tomli=2.0.1=pyhd8ed1ab_0
+  - tornado=6.3.2=py310h2372a71_0
+  - tqdm=4.65.0=pyhd8ed1ab_1
+  - traitlets=5.9.0=pyhd8ed1ab_0
+  - typing-extensions=4.7.1=hd8ed1ab_0
+  - typing_extensions=4.7.1=pyha770c72_0
+  - typing_utils=0.1.0=pyhd8ed1ab_0
+  - tzdata=2023c=h71feb2d_0
+  - umap-learn=0.5.3=py310hff52083_1
+  - unicodedata2=15.0.0=py310h5764c6d_0
+  - urllib3=1.26.15=pyhd8ed1ab_0
+  - wcwidth=0.2.6=pyhd8ed1ab_0
+  - webencodings=0.5.1=py_1
+  - websocket-client=1.6.1=pyhd8ed1ab_0
+  - wheel=0.40.0=pyhd8ed1ab_0
+  - widgetsnbextension=4.0.8=pyhd8ed1ab_0
+  - xcb-util=0.4.0=hd590300_1
+  - xcb-util-image=0.4.0=h8ee46fc_1
+  - xcb-util-keysyms=0.4.0=h8ee46fc_1
+  - xcb-util-renderutil=0.3.9=hd590300_1
+  - xcb-util-wm=0.4.1=h8ee46fc_1
+  - xkeyboard-config=2.39=hd590300_0
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.1.1=hd590300_0
+  - xorg-libsm=1.2.4=h7391055_0
+  - xorg-libx11=1.8.6=h8ee46fc_0
+  - xorg-libxau=1.0.11=hd590300_0
+  - xorg-libxdmcp=1.1.3=h7f98852_0
+  - xorg-libxext=1.3.4=h0b41bf4_2
+  - xorg-libxrender=0.9.11=hd590300_0
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h0b41bf4_1003
+  - xorg-xf86vidmodeproto=2.3.1=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xz=5.2.6=h166bdaf_0
+  - yaml=0.2.5=h7f98852_2
+  - zeromq=4.3.4=h9c3ff4c_1
+  - zipp=3.15.0=pyhd8ed1ab_0
+  - zlib=1.2.13=hd590300_5
+  - zstd=1.5.2=hfc55251_7
   - pip:
-    - git+https://github.com/d33bs/cytotable@c6e0046e76986f9aa837b441186b621fe06b8189
-    - duckdb==0.8.0
+      - azure-core==1.27.1
+      - azure-storage-blob==12.16.0
+      - bcrypt==4.0.1
+      - cachetools==5.3.1
+      - cfgv==3.3.1
+      - cloudpathlib==0.13.0
+      - cytotable==0.0.1
+      - dill==0.3.6
+      - distlib==0.3.6
+      - duckdb==0.7.1
+      - filelock==3.12.2
+      - globus-sdk==3.23.0
+      - google-api-core==2.11.1
+      - google-auth==2.21.0
+      - google-cloud-core==2.3.3
+      - google-cloud-storage==2.10.0
+      - google-crc32c==1.5.0
+      - google-resumable-media==2.5.0
+      - googleapis-common-protos==1.59.1
+      - identify==2.5.24
+      - isodate==0.6.1
+      - nodeenv==1.8.0
+      - paramiko==3.2.0
+      - parsl==2023.7.3
+      - pre-commit==3.3.3
+      - protobuf==4.23.4
+      - pyarrow==11.0.0
+      - pyasn1==0.5.0
+      - pyasn1-modules==0.3.0
+      - pycytominer==0.2.0
+      - pyjwt==2.7.0
+      - pynacl==1.5.0
+      - rsa==4.9
+      - setproctitle==1.3.2
+      - sqlalchemy==1.4.49
+      - tblib==2.0.0
+      - typeguard==2.13.3
+      - types-paramiko==3.2.0.0
+      - types-requests==2.31.0.1
+      - types-six==1.16.21.8
+      - types-urllib3==1.26.25.13
+      - virtualenv==20.23.1
+prefix: /home/camo/anaconda3/envs/jump_sc
```

Comment:
what is `prefix`? Note, this conda environment should work for other computers and users

Reply:
I think the `prefix` is a consequence of exporting the environment. Please see the comment above.